recipe1.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Grilled Chicken & Salad</title>
+    <link rel="stylesheet" href = "index.css">
+</head>
+<header>
+    <div class = "header">
+        <h1> Coolio Cooking </h1>
+        <br>
+        <h2> <a href = "index.html">Home</a> </h2>
+        <h2> <a href = "aboutme.html">About Me</a> </h2>
+    </div>  
+</header>
+<main>
+    <div class = "page-of-recipe1">
+        <h2> <a class = "grilled_chk">Grilled Chicken & Salad</a> </h2>
+        <div class = "recipe1_subpage">
```

Comment:
I would suggest making your classes more distinct in name and following a convention (lowercase for html classes and ids delimited by a dash so `recipe-1-subpage` would be appropriate), both so it's easier to distinguish and read!